### ISSUE TITLE
About the accuracy

### ISSUE THREAD

**Issue by starhiking** [apchenstu/mvsnerf#17]
Hi,  I have some questions concerning accuracy in the following:

1. How to evaluate the accuracy for the 16 DTU?  I have changed the `max_len` from 10 to -1 in val_dataset. Does the modification work right and Does there exist other methods for evaluating? 

2. Except #114 scene in DTU,  I cannot implement the accuracy of other scenes (1, 8, 21, 103) as the paper said.
Could you please offer the run commands for pretraining and fine-tuning (DTU) to the official accuracy?
And I am sure the dataset is right because the #114 also comes from it.

3. By the way, is the newest update(597f5) significant,  can the older version (736d9a4b) be restored its accuracy?

**Comment by starhiking** [comment]
And how to set the `netwidth`?

**Comment by apchenstu** [comment]
we have provided the training and fine-tuning script at [run_batch.py](https://github.com/apchenstu/mvsnerf/blob/main/run_batch.py), also, the rendering and quantity script located in 'renderer.ipynb', you can reproduce our acc with the scripts. Before that, you have better pull down the newest version.
